utils/forest_utils/src/io/progress_bar.rs

```diff
@@ -42,47 +42,63 @@ static PROGRESS_BAR_VISIBILITY: RwLock<ProgressBarVisibility> =
 
 /// Progress bar wrapper, allows suppressing progress bars.
 pub struct ProgressBar {
-    inner: RefCell<pbr::ProgressBar<Stdout>>,
+    inner: RefCell<IndicatifProgressBar>,
     display: bool,
+    size: u64,
 }
 
 impl ProgressBar {
     pub fn new(size: u64) -> Self {
         Self {
-            inner: RefCell::new(pbr::ProgressBar::new(size)),
+            inner: RefCell::new(IndicatifProgressBar::with_draw_target(
+                Some(size),
+                ProgressDrawTarget::stdout(),
+            )),
             display: Self::should_display(),
+            size,
         }
     }
 
     pub fn message(&self, message: &str) {
         if self.display {
-            self.inner.borrow_mut().message(message);
+            self.inner.borrow_mut().set_message(message.to_string());
         }
     }
 
-    pub fn set_max_refresh_rate(&self, w: Option<Duration>) {
+    pub fn set_max_refresh_rate(&mut self, w: u8) {
```

Comment:
It's not obvious to me that `w` is given in hertz. Maybe we could change the name of the function to `set_max_refresh_rate_in_hz`.

Reply:
Ok. But it's actually true. Internally indicatif really divides 1000 ms by this number

Reply:
Fixed